components/brave_rewards/resources/shared/lib/external_wallet.ts

```diff
@@ -99,7 +99,7 @@ export function externalWalletFromExtensionData (
     links: {
       account: String(data.accountUrl || ''),
       addFunds: String(data.addUrl || ''),
-      completeVerification: String(data.verifyUrl || ''),
+      completeVerification: String(data.loginUrl || ''),
```

Comment:
I'm thinking that we can probably get rid of this field entirely, and also the `complete-verification` option in `ExternalWalletAction` (from `external_wallet_action.ts`).

Reply:
Yeah, that's what I was thinking, too. Wanted to reach out to you to discuss. 🙂

Reply:
Resolved in 6379b0a4c4ace23910d8dd2694aaf9ae8a335426.